data/records.json

```
@@ -125,4 +125,25 @@
         "license": "freeware",
         "upload_date": "12/11/2021"
     }
+    {
+        "name": "SpotiBot - Spotify Automation",
+        "description_short": "The bot makes automatic login in spotify, mutes ads and tries to skip them",
+        "description": "SpotiBot is a automation for muting and skipping ads. You can use it as template to try to create your own spotify automation. Makes automatically login before starts to muting",
+        "tags": ["Spotify", "automation","WebBot", "web","bot"],
+        "programming_language": "python",
+        "technology": ["botcity web framework"],
+        "OS": ["windows"],
+        "thumbnail_url": "https://imgur.com/a/Iys0UG4",
+        "youtube_video": "https://youtu.be/YF3BQoyegN0",
+        "youtube_width": 1280,
+        "youtube_height": 720,
+        "author_name": "Victor 'VeK' Ferreira",
+        "author_site": "https://github.com/ProjektVek",
+        "repository_name": "Spotify-Automation",
+        "repository_url": "https://github.com/ProjektVek",

```

The repo URL should also include the repository name:
```suggestion
        "repository_url": "https://github.com/ProjektVek/Spotify-Automation",
```